### PR TITLE
Prom grafana

### PR DESCRIPTION
**Pull request by ngmachado**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,22 +1,69 @@
...
+      container_name: grafana
+      volumes:
+        - grafana_data:/var/lib/grafana
+        - ./grafana/provisioning:/etc/grafana/provisioning
+        - ./grafana/dashboards/:/etc/grafana/dashboards
+        - ./grafana/grafana.ini:/etc/grafana/grafana.ini
+      environment:
+        - GF_AUTH_DISABLE_LOGIN_FORM=true
+        - GF_AUTH_ANONYMOUS_ENABLED=true
+        - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
```
Comment (d10r) [inline docker-compose.yml:57]: While useful for testing, I don't think we should ship with this config.  
By default it requires login, with credentials preset to admin:admin and a forced pw change on first login. I think that's sane defaults and suggest to remove this env vars before merging.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,22 +1,69 @@
...
+      ports:
+        - ${GRAFANA_PORT:-3000}:3000
+      expose:
+        - ${GRAFANA_PORT:-3000}
+      networks:
+        - monitoring
 
 volumes:
+  prometheus_data: { }
+  grafana_data: { }
```
Comment (d10r) [inline docker-compose.yml:67]: what does the value { } mean here? Is it different from no value like for `data` ?

**Comment by d10r** [comment]
1.  As far as I see, no Grafana dashboard is included yet, right?
2.  It seems to me that we currently have no update path. Once the sentinel image was built, it never seems to be updated unless the container and image are manually deleted. How can we "fix" that?
3. Related to 2: while I like the possibility to easily build a container from scratch, this may not be ideal for running on resource constrained devices. How about fetching the image from Docker hub by default and offer local building as an option (e.g. different docker-compose file)? In that case it would imo also be ok if updating requires manual deletion of the image.

File: docker-compose.yml (hunk 2 above)
Comment (ngmachado) [reply docker-compose.yml:67]: No, is the same.

File: docker-compose.yml (hunk 1 above)
Comment (ngmachado) [reply docker-compose.yml]: Removed as suggested

**Comment by ngmachado** [comment]
This PR was outdated by multi smaller PR.

**Pull request closed without merging**